packages/engine/paima-sm/src/types.ts

```diff
@@ -262,6 +288,20 @@ export type ChainDataExtensionCardanoDelegation = ChainDataExtensionBase &
     cdeType: ChainDataExtensionType.CardanoPool;
   };
 
+export const ChainDataExtensionCardanoProjectedNFTConfig = Type.Object({
+  type: Type.Literal(CdeEntryTypeName.CardanoProjectedNFT),
+  contract: Type.String(),
```

Comment:
Probably this should be more description (ex: `scriptHash`)

Reply:
this is unused anyway now